web/src/components/tasks/RaffleBingoPaginator.tsx

```diff
@@ -0,0 +1,175 @@
+import { Content } from "@/components/Content";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { CtaContainer } from "@/components/njwds-extended/cta/CtaContainer";
+import { SingleCtaLink } from "@/components/njwds-extended/cta/SingleCtaLink";
+import { HorizontalStepper } from "@/components/njwds-extended/HorizontalStepper";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { ActionBarLayout } from "@/components/njwds-layout/ActionBarLayout";
+import {
+  getRaffleBingoStep,
+  isLastStep,
+  isNotFirstStep,
+  isNotLastStep,
+  raffleBingoStepFiles,
+} from "@/components/tasks/RaffleBingoSteps";
+import { TaskStatusChangeSnackbar } from "@/components/TaskStatusChangeSnackbar";
+import { fetchTaskByFilename } from "@/lib/async-content-fetchers/fetchTaskByFilename";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { StepperStep, Task } from "@/lib/types/types";
+import { TaskProgress } from "@businessnjgovnavigator/shared/userData";
+import { ReactElement, ReactNode, useEffect, useState } from "react";
+
+interface Props {
+  task: Task;
+  CMS_ONLY_STEPINDEX?: number; // CMS only
```

Comment:
[sand] Comment is redundant.

I'd also make `STEPINDEX` the same case as the variable name we're overriding, but that's just me.
```suggestion
  CMS_ONLY_stepIndex?: number;
```